cpp/pybind/t/pipelines/odometry/odometry.cpp

```diff
@@ -0,0 +1,275 @@
+                 "by CreateVertexMap before calling this function."},
+                {"target", "The target RGBD image."},
+                {"target_depth",
+                 "(row, col, channel = 1) Float32 target depth image obtained "
+                 "by PreprocessDepth before calling this function."},
+                {"target_depth_dx",
+                 "(row, col, channel = 1) Float32 target depth gradient image "
+                 "along x-axis obtained by FilterSobel before calling this "
+                 "function."},
+                {"target_depth_dy",
+                 "(row, col, channel = 1) Float32 target depth gradient image "
+                 "along y-axis obtained by FilterSobel before calling this "
+                 "function."},
+                {"target_intensity",
+                 "(row, col, channel = 1) Float32 target intensity image "
+                 "obtained by RGBToGray before calling this function"},
+                {"target_intensity_dx",
+                 "(row, col, channel = 1) Float32 target intensity gradient "
+                 "image along x-axis obtained by FilterSobel before calling "
+                 "this function."},
+                {"target_intensity_dy",
+                 "(row, col, channel = 1) Float32 target intensity gradient "
+                 "image along y-axis obtained by FilterSobel before calling "
+                 "this function."},
+                {"target_normal_map",
+                 "(row, col, channel = 3) Float32 target normal image obtained "
+                 "by CreateNormalMap before calling this function."},
+                {"target_vertex_map",
+                 "(row, col, channel = 3) Float32 target vertex image obtained "
+                 "by CreateVertexMap before calling this function."}};
+
+void pybind_odometry_methods(py::module &m) {
+    m.def("rgbd_odometry_multi_scale", &RGBDOdometryMultiScale,
+          "Function for Multi Scale RGBD odometry.", "source"_a, "target"_a,
+          "intrinsics"_a,
+          "init_source_to_target"_a = core::Tensor::Eye(4, core::Dtype::Float64,
+                                                        core::Device("CPU:0")),
+          "depth_scale"_a = 1000.0f, "depth_max"_a = 3.0f,
+          "criteria_list"_a =
+                  std::vector<OdometryConvergenceCriteria>({10, 5, 3}),
+          "method"_a = Method::Hybrid, "params"_a = OdometryLossParams());
+    docstring::FunctionDocInject(m, "rgbd_odometry_multi_scale",
+                                 map_shared_argument_docstrings);
+
+    m.def("compute_odometry_result_point_to_plane",
+          &ComputeOdometryResultPointToPlane,
+          "(Estimates the OdometryResult (4x4 rigid transformation "
+          "T from source to target, with inlier rmse and fitness). "
+          "Performs one iteration of RGBD odometry using "
+          "Loss function: :math: `[(V_p - V_q)^T N_p]^2`, where "
```

Comment:
@ssheorey I tried, but it doesn't seem to work. May you please help me with this. 
I referred to the legacy `odometry` and `trainglemesh` pybinds, but it is still not fixed.

Reply:
Done.